launcher/screens/Guide/Dashboard/BattlefieldOverview/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Image, StyleSheet, View } from 'react-native';
+import { DimensionState, dimensionState, Text } from '@metacraft/ui';
+import { battlefield } from 'screens/Guide/content';
+import ConceptButton from 'screens/Guide/Dashboard/ConceptButton';
+import { sharedStyle } from 'screens/Guide/shared';
+import { useSnapshot } from 'utils/hook';
+import resources from 'utils/resources';
+
+const styles = StyleSheet.create({
+	conceptContainer: {
+		flexDirection: 'row',
+		flexWrap: 'wrap',
+		justifyContent: 'space-evenly',
+		marginTop: 40,
+	},
+});
+
+const BattlefieldOverview: React.FC<Record<string, unknown>> = () => {
+	const [selectedConcept, setSelectedConcept] = React.useState<number>(0);
```

Comment:
IMO, `import { FC, useState }` is less verbose, improving readability, less typing and clean code